<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>File Transfer Server :: Board</string>
  </property>
  <property name="windowIcon">
   <iconset resource="application.qrc">
    <normaloff>:/Resources/minertocat.png</normaloff>:/Resources/minertocat.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>450</width>
      <height>155</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Simplified Arabic Fixed</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Information</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>35</y>
       <width>190</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Simplified Arabic Fixed</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Thread Number</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLCDNumber" name="LCD_ThreadNumber">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>45</y>
       <width>90</width>
       <height>35</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>85</y>
       <width>190</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Simplified Arabic Fixed</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Connection Number</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLCDNumber" name="LCD_ConnectionNumber">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>95</y>
       <width>90</width>
       <height>35</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>450</width>
      <height>311</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Simplified Arabic Fixed</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Clients</string>
    </property>
    <widget class="QListView" name="ListView_ClientList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>431</width>
       <height>281</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>40</y>
      <width>451</width>
      <height>491</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Simplified Arabic Fixed</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Logs</string>
    </property>
    <widget class="QListView" name="ListView_Logs">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>431</width>
       <height>461</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="application.qrc"/>
 </resources>
 <connections/>
</ui>
